sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<p>General Metrics Config does not exist anymore. We need to replace <code>general_metrics_config</code>  with <code>python_profiling_config</code>  everywhere in this notebook and update <code>GeneralMetricsConfig</code>  with <code>PythonProfilingConfig</code>  here. Also need to update <code>use_pyinstrument</code>  with something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">use_pyinstrument = python_profiling_config["ProfilerName"] == "pyinstrument"
</pre><p> </p><p>Alternate solution (if we want to make the notebook look more clean): we can directly access the sagemaker SDK <code>PythonProfilingConfig</code> object that was defined earlier for the profiler config in the estimator. Now we simply rely on this object and not the dictionary structure (which remains a black box for users).</p><p> </p><pre class="ql-syntax" spellcheck="false">python_profiling_config = PythonProfilingConfig(
    start_step=9,&nbsp;
    num_steps=1
)

...

profiler_config = ProfilerConfig(
    ...
    framework_profile_params=FrameworkProfileParams(
        ...
        python_profiling_config=python_profiling_config
        ...
    )
)

...

use_pyinstrument = python_profiling_config.python_profiler == PythonProfiler.PYINSTRUMENT
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>fixed, and moved this section to Python profiling section below.</p>